wpeverest/user-registration
# Addon - Frontend listing

### All Submissions:

* [x] Have you checked to ensure there aren't other open [Pull Requests](../pull) for the same update/change?





### Changes proposed in this Pull Request:
This PR provides support for the new User Registration Frontend Listing Addon.

### How to test the changes in this Pull Request:
1. Install and activate User Registration Frontend Listing Addon.
2. Switch to this branch.
3. Check whether all the functionality of the frontend listing addon is working properly.

### Types of changes:




### Other information:




### Changelog entry

> Addon - Frontend listing
